### ISSUE TITLE
vfold_cv / initial split with id option for repeated measures / panel designs

### ISSUE THREAD

**Issue by 16mc1r** [tidymodels/rsample#260]
Hello, coming from caret starting in tidymodels: 

I work with truly massive panel data (so low frequency is not an issue) with repeated measures for each person. Were talking about hundrets of datapoints each year over more than a decade.

My issue: If I split the data it splits along rows, not simply sampling the ids. Thus some ids can end up in training and test simultaneously and only a portion of the data for each selected id is used. 

I can do a workaround by sampling only the ids and then constructing a split object as in #109. That works for one instance but what about vfold_cv? It seems an awful hassle to construct 10 fold cv sets with the make_split integer method.

**Comment by juliasilge** [comment]
It sounds to me like this is related to #207 

Have you looked at using [`group_vfold_cv()`](https://rsample.tidymodels.org/reference/group_vfold_cv.html)?

**Comment by 16mc1r** [comment]
Thanks, `group_vfold_cv` does work, though it would be great if there was an option to control the proportion of test to training set within the splits directly via a `prop` option. Currently its done indirecty through the number of folds.

So to include stratification by a variable that is separate from id i simply create a composite id like `glue({id}_{strata_1})` ? 

If that is correct I suggest updating the `group_vfold_cv` parameter description for `group`
from the generic : `This could be a single character value or a variable name that corresponds to a variable that exists in the data frame` to something along the lines: 

>group: Grouping variable that is used to assign all rows of the same grouping level (i.e. same value for group) to either the training or the test set. For example, assigning multiple values of the same subject to either training or test.

**Comment by juliasilge** [comment]
Yep, that is how I would currently recommend both "stratifying" and grouping by multiple variables in your dataset.

On your first point, maybe I could use more clarification. With cross validation, you can't really specify `prop` per se; you set `v`, or in the case of groups like here, groups + `v`. Do you mean you are interested in grouped `initial_split()`, as described in #207?
